src/Core/TileMesh.js

```diff
@@ -49,23 +49,47 @@ class TileMesh extends THREE.Mesh {
      * If specified, update the min and max elevation of the OBB
      * and updates accordingly the bounding sphere and the geometric error
      *
-     * @param {?number} min
-     * @param {?number} max
-     * @param {?number} scale
+     * @param {Object}  elevation
+     * @param {?number} elevation.min
+     * @param {?number} elevation.max
+     * @param {?number} elevation.scale
+     * @param {?number} elevation.geoidHeight
      */
-    setBBoxZ(min, max, scale) {
-        if (min == null && max == null) {
-            return;
+    setBBoxZ(elevation) {
+        if (arguments['1']) {  // eslint-disable-line prefer-rest-params
+            console.warn(
+                '`TileMesh.setBBox` usage with `min, max, scale` parameters is deprecated. It should now be ' +
+                'used with `{ min, max, scale, geoidHeight }` object instead. See documentation for more information.',
+            );
+            elevation = {
+                min: arguments['0'],  // eslint-disable-line prefer-rest-params
+                max: arguments['1'],  // eslint-disable-line prefer-rest-params
+                scale: arguments['2'],  // eslint-disable-line prefer-rest-params
+            };
         }
```

Comment:
could you remove this warning and add a `BREAKING CHANGES`

Reply:
I removed the warning.